libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -67,6 +67,9 @@ export const HOST_REQUIREMENTS_LINK = 'https://access.redhat.com/solutions/48856
 export const ODF_REQUIREMENTS_LINK =
   'https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation';
 
+export const OSC_REQUIREMENTS_LINK =
+  'https://www.redhat.com/en/blog/red-hat-openshift-confidential-containers-deployment-solution';
+
```

Comment:
@jhernand How about using the https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.8/html/user_guide/deploying-osc-bare-metal#osc-resource-requirements_deploying-bare-metal link here? Thanks.

Reply:
Sure, this was just an example. I will do that change and rebase.

Reply:
Done.